File: C LANGUAGE/19.c
// 19. Check if the array arr = [1, 2, 3, 4, 5] is sorted in ascending order, and print true or false.

#include<stdio.h>
int main(){
    int arr[] ={1,2,3,4,5};
      int length = sizeof(arr)/sizeof(arr[0]);
      int i = 0;
      int count = 0;

      for(i = 0;i<length-1;i++){
        if(arr[i]>arr[i+1]){
            count ++ ;
        }
      }
      if(count == 0){
        printf("true");
      }else{
        printf("flase");
      }
}

File: JAVASCRIPT/19.js
// 19. Check if the array arr = [1, 2, 3, 4, 5] is sorted in ascending order, and print true or false.

var array = [1,2,3,4,5];

var count = 0;
var i =0;

for(i =0 ; i<array.length ; i++){
    if(array[i] > array[i +1]){
        count ++ ;
    }
}

if(count == 0 ){
    console.log("true");
}else{
    console.log("false");
}
